doc/stdlib/math.rst

```diff
@@ -1396,47 +1396,53 @@ Mathematical Functions
 
    Compute the polygamma function of order ``m`` of argument ``x`` (the ``(m+1)th`` derivative of the logarithm of ``gamma(x)``\ )
 
-.. function:: airy(k,x)
+.. function:: airyai(x)
```

Comment:
the contents don't match https://github.com/JuliaLang/julia/blob/54ec2150afa6ba03344d08a7ab85be89fe265e17/base/special/bessel.jl#L43-L46